src/biokbase/narrative/tests/test_system.py

```diff
@@ -0,0 +1,90 @@
+import os
```

Comment:
Redid the tests here as `pytest` style, since we're using `pytest` anyway and it would be nice to migrate there.